content/en/docs/how-to/registry-authentication.md

```diff
@@ -93,10 +73,46 @@ unset NOTATION_USERNAME
 unset NOTATION_PASSWORD
 ```
 
+## Configure `config.json` to authenticate to an OCI-compliant registry
+
+{{% alert title="Warning" color="warning" %}}
+Storing credentials in `config.json` is not secure for storing sensitive data such as usernames and passwords and should not be used in a production environment.
+{{% /alert %}}
+
+You can configure the [`notation` config.json file]({{< ref "/docs/concepts/directory-structure" >}}) to store the credentials for your OCI-compliant registry as base64 encoded values. For example:
+
+```json
+{
+  "auths": {
+    "<registry>": {
+      "auth": "<base64encoded>",
+    }
+  },
+}
+```
+
+## Use `notation login` to authenticate to an OCI-compliant registry
+
+To authenticate to an OCI-compliant registry, use the `notation login` command with your registry, username, and password. For example:
+
+```console
+notation login -u <username> -p <password> <registry>
+```
+
+{{% alert title="Note" color="primary" %}}
+If `notation login` is failing, you may need to [Configure a credentials store](#configure-a-credentials-store). Alternatively in development and testing environments, you can use [environment variables](#configure-environment-variables-to-authenticate-to-an-oci-compliant-registry) to authenticate to an OCI-compliant registry.
+{{% /alert %}}
+
 ## Use `notation logout` to log out of an OCI-compliant registry
 
 To log out of an OCI-compliant registry, use the `notation logout` command with your registry. For example:
 
+{{% alert title="Warning" color="warning" %}}
+Using `notation logout` will remove the credentials from the configured credentials store. If you are using the Docker configuration file as your credentials store, `notation logout` will remove the credentials from the Docker configuration file.
+
+If both the Docker configuration file and the `notation` configuration file are configured with a credentials store, `notation logout` will remove the credentials from credentials stored configured in the `notation` configuration file.
```

Comment:
This behavior doesn't seem to be deterministic. I think when users run `notation logout`, all related credentials should be removed? /cc: @shizhMSFT @yizha1 @FeynmanZhou